flexget/utils/serialization.py

```diff
@@ -140,11 +135,11 @@ def serializer_handles(cls, value):
         return isinstance(value, set)
 
     @classmethod
-    def serialize(cls, value: set):
+    def serialize(cls, value: Set[T]) -> List[T]:
```

Comment:
Not sure it matters, but I don't think this is strictly true. This function will take a set of T, but then return a list of the serialized form of T. Probably easier to just loosen these definitions and skip the `T`

Reply:
You are correct - generics were removed from annotations.